store_psql.py

```diff
@@ -0,0 +1,121 @@
+system_id                                   bigint not null,
+"addition temperature BT63"                 float,
+"avg. outdoor temp. BT1"                    float,
+"blocked"                                   boolean,
+"calculated flow temp. S1"                  float,
+"comfort mode"                              text,
+"compressor operating time EB101"           float,
+"compressor operating time hot water EB101" float,
+"compressor run time cooling EB101"         float,
+"compressor starts EB101"                   text,
+"condenser out EB101-BT12"                  float,
+"cooling, compr. only."                     float,
+"cpr. protection mode EB101"                boolean,
+"current BE1"                               float,
+"current BE2"                               float,
+"current BE3"                               float,
+"current compr. frequency EB101"            integer,
+"defrosting EB101"                          boolean,
+"degree minutes"                            integer,
+"electrical addition power"                 float,
+"evaporator EB101-BT16"                     float,
+"external adjustment S1"                    boolean,
+"floor drying function"                     boolean,
+"flow BF1"                                  float,
+"fuse size"                                 float,
+"heat medium flow BT2"                      float,
+"heating, compr. only."                     float,
+"heating, int. add. incl."                  float,
+"high pressure sensor EB101"                float,
+"hot gas EB101-BT14"                        float,
+"hot water charging BT6"                    float,
+"hot water top BT7"                         float,
+"hotwater, compr. only."                    float,
+"hw, incl. int. add"                        float,
+"liquid line EB101-BT15"                    float,
+"low pressure sensor EB101"                 float,
+"outdoor temp. BT1"                         float,
+"outdoor temp. EB101-BT28"                  float,
+"pool, compr. only."                        float,
+"pump speed heating medium GP1"             integer,
+"requested compressor freq EB101"           integer,
+"return temp. BT3"                          float,
+"return temp. EB101-BT3"                    float,
+"set max electrical add."                   float,
+"suction gas EB101-BT17"                    float,
+"temporary lux"                             boolean,
+"time factor"                               float,
+"external flow temp. BT25"                  float,
+"room temperature BT50"                     float,
+"tehowatti addition blocked"                boolean,
+"inverter m8 compressor module blocked"     boolean,
+"test"                                      integer
+)
+'''
+
+# map duplicate measurement headers to separate columns
+column_mapping = {
+    "10033": "tehowatti addition blocked",
+    "10014": "inverter m8 compressor module blocked"
```

Comment:
Should we just change the column names in question so we don't need this mapping?